prometheus_client/metrics_core.py

```diff
@@ -236,6 +236,7 @@ def __init__(self,
                  sum_value: Optional[float] = None,
                  labels: Optional[Sequence[str]] = None,
                  unit: str = '',
+                 native_hist_bucket_factor: Optional[float] = None
```

Comment:
🤔 Not sure if we need this here at all (it will need to be in `metrics.py`). This function is for custom collectors, and I think if someone is creating a native histogram for that case they will end up needing to define all the spans themself. For now I would say just leave it out.

Reply:
Yes, I deleted it.